lib/Models/Leaflet.ts

```diff
@@ -48,6 +47,9 @@ import MapInteractionMode from "./MapInteractionMode";
 import i18next from "i18next";
 import ImageryProvider from "terriajs-cesium/Source/Scene/ImageryProvider";
 
+const FeatureDetection: FeatureDetection = require("terriajs-cesium/Source/Core/FeatureDetection")
```

Comment:
So this one is a bit of an interesting case like the one with `Scene` (also `Axis`), where Kevin did a declaration merge but it threw an error. Essentially I had to make it an interface type thing. If I use import it doesn't look at the interface declared in `index.d.ts` at all, and instead throws the error `Property 'isInternetExplorer' does not exist on type 'typeof FeatureDetection'` because the module in `cesium.d.ts` doesn't have that function/property for some reasons? Using `require` + type casting works. I had to make sure it doesn't lose the type checking by casting it to the interface declared in `index.d.ts`

Just noting that I added the interface declaration in `lib/ThirdParty/terriajs-cesium-extra/index.d.ts` because it's used in more than 1 file.

Reply:
Right, gotcha. Could you please stick a comment in the code next to the places you do this to explain?

Reply:
Yup, will do!